roamance/src/components/profile/profile-header.tsx

```
@@ -0,0 +1,232 @@
+
+export function ProfileHeader({
+  user,
+  loading,
+  onProfileUpdate,
+}: ProfileHeaderProps) {
+  const router = useRouter();
+  const [isUploading, setIsUploading] = useState(false);
+  const [isUploadDialogOpen, setIsUploadDialogOpen] = useState(false);
+
+  // Calculate initials for avatar fallback
+  const getInitials = (name: string = '') => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)